src/kokkos_abstraction.hpp

```diff
@@ -52,6 +53,26 @@ using ParArray5D = Kokkos::View<T *****, LayoutWrapper, DevMemSpace>;
 template <typename T>
 using ParArray6D = Kokkos::View<T ******, LayoutWrapper, DevMemSpace>;
 
+template <bool>
+struct UseLightweightKernel {};
+
+template <typename PolicyType>
+auto par_policy(PolicyType const &policy_old, UseLightweightKernel<true>) {
+  auto policy_new = Kokkos::Experimental::require(
+      policy_old, Kokkos::Experimental::WorkItemProperty::HintLightWeight);
+  return policy_new;
+}
```

Comment:
It would perhaps be worth documenting what the lightweight version does?

Reply:
Yes, (though I'd consider this feature still experimental)